### PR TITLE
Raise upper bound of analyzer dependency

### PR DESCRIPTION
**Pull request by greglittlefield-wf**
QA instructions:
- Verify that `analyzer` 0.39.x was pulled into CI

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/analyzer_helpers.dart [hunk 1]
```diff
@@ -180,8 +180,6 @@ String _copyFieldDeclaration(FieldDeclaration decl, String initializer) {
 }
 
 String _copyGetterDeclaration(MethodDeclaration decl, String body) {
-  bool isAsync =
-      (decl.returnType?.type?.name ?? decl.returnType?.toString()) == 'Future';
```
Comment (greglittlefield-wf) [inline lib/src/analyzer_helpers.dart]: `DartType.name` was deprecated, so this had to be changed. The `keyword` solution below should be more robust, anyways

**Comment by aaronlademann-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
+1 security
- Change to watched file is to update an import, but is functionally equivalent

**Comment by aaronlademann-wf** [comment]
@Workiva/release-management-pp

**Pull request merged**